### PR TITLE
Custom skyboxes

### PR DESCRIPTION
**Pull request by xomarko**

### CODE REVIEW

File: Osiris/Config.cpp [hunk 1]
```diff
@@ -437,6 +437,7 @@ static void from_json(const json& j, Config::Visuals& v)
     read(j, "Far Z", v.farZ);
     read(j, "Flash reduction", v.flashReduction);
     read(j, "Skybox", v.skybox);
+    read<value_t::object>(j, "Custom skybox", v.customSkybox);
```
Comment (XJle6uIIIeK) [inline Osiris/Config.cpp:440]: Use 
read<value_t::string>(j, "Custom skybox", v.customSkybox);

**Comment by xomarko** [comment]
> > 
> 
> This is actually something all cheat needs

agreed

**Comment by Navoichyk** [comment]
> > > 
> > 
> > 
> > This is actually something all cheat needs
> 
> agreed

what do u enter to load the skybox? It might just be the skyboxes I found but nothing is working.

**Comment by xomarko** [comment]
> > > > 
> > > 
> > > 
> > > This is actually something all cheat needs
> > 
> > 
> > agreed
> 
> what do u enter to load the skybox? It might just be the skyboxes I found but nothing is working.

u enter the skybox filename

**Comment by Xsintashi** [comment]
> 

U have to put ur skyboxes in `csgo\materials\skybox`. Then in input in game type skybox name (Remeber to remove 2 letters in input cuz those letters shows direction of skybox). Sorry for bad explanation.

**Comment by xomarko** [comment]
image

**Comment by Navoichyk** [comment]
i couldn’t get it to work. are the files supposed to be vmts?

**Comment by xomarko** [comment]
> i couldn’t get it to work. are the files supposed to be vmts and stuff?

yep

**Comment by Xsintashi** [comment]
Imma just record it. Will be easier

**Pull request merged**